examples/1.blockade/main.jl

```diff
@@ -199,17 +207,36 @@
 # observing the checkerboard phase (nearest-neighbor blockade) and the striated phase (next-nearest-neighbor blockade)
 # as in this [paper](https://www.nature.com/articles/s41586-021-03582-4),
 # following the same rationale as above.
-
-# For an example arbitrary graph shown in the right most panel of the above figure,
+# Here, we summarize the recommended lattice constants (unit μm) for common lattices in the following table:
```

Comment:
just a thought maybe we should have this table programmatically in the package, so one can directly use it - not sure what's the best way to have it tho

Reply:
yes - let's discuss that when you are here next week!